lib/src/state_manager/rx/rx_impl.dart

```diff
@@ -442,24 +474,109 @@ class RxDouble extends _RxImpl<double> {
   RxDouble([double initial]) {
     _value = initial;
   }
+
+  RxDouble operator +(double val) {
+    subject.add(value += val);
+    return this;
+  }
+
+  RxDouble operator -(double val) {
+    subject.add(value -= val);
+    return this;
+  }
+
+  RxDouble operator /(double val) {
+    subject.add(value /= val);
+    return this;
+  }
+
+  RxDouble operator *(double val) {
+    subject.add(value *= val);
+    return this;
+  }
 }
 
 class RxNum extends _RxImpl<num> {
   RxNum([num initial]) {
     _value = initial;
   }
+
+  RxNum operator >>(num val) {
```

Comment:
i'm sorry, this question is problably very silly, but i'll ask anyway:
I case i want to actually use this operator for bitwise operations, this will interfer somehow? Sorry i am asking because I truly don't know

Reply:
Apparently, this operator is being used as a setter, to avoid using the .value

Reply:
If u wanna use that operator will be only valid for int types, mostly to shift hex values... is pretty uncommon and kinda low level. In any case if you need to do that in a RxInt just access the rx.value >> and will work on the actual int instance. For all Rx types this operator was choosen as "assignment"
So if u do
final msg="hi".obs
msg>>"goodbye"
That, even without the >>=, will assign the value and add it to the stream.